scalding-core/src/main/scala/com/twitter/scalding/Job.scala

```diff
@@ -77,6 +77,7 @@ object Job {
  * these functions can be combined Monadically using algebird.monad.Reader.
  */
 class Job(val args: Args) extends FieldConversions with java.io.Serializable {
+  Tracing.init()
```

Comment:
Doesn't seem like it should be in the default job, something can be opted into by extending a trait instead